optimas/diagnostics/exploration_diagnostics.py

```diff
@@ -321,3 +523,41 @@ def plot_worker_timeline(
 
         ax.set_ylabel("Worker")
         ax.set_xlabel("Time (s)")
+
+    def _check_pareto_objectives(
+        self,
+        objectives: Optional[List[Union[str, Objective]]] = None,
+    ) -> List[Objective]:
+        """Check the objectives given to the Pareto methods."""
+        if len(self.objectives) < 2:
+            raise ValueError(
+                "Cannot get Pareto front because only a single objective "
+                "is available."
+            )
+        if objectives is None:
+            if len(self.objectives) == 2:
```

Comment:
If not specified, it will work only if two objectives are found in the exploration folder.
It could work as well when there are more than two objectives by selecting the first two of them by default:
```py
if len(self.objectives) >= 2:
    objectives = self.objectives[:2]
```